src/modules/hip/kernel/crop_mirror_normalize.hpp

```diff
@@ -69,15 +79,14 @@ __global__ void crop_mirror_normalize_pkd_tensor(T *srcPtr,
         rpp_hip_load24_pkd3_and_unpack_to_float24_pln3(srcPtr + srcIdx, &pix_f24);
     }
 
-    uint dstIdx = (id_z * dstStridesNH.x) + (id_y * dstStridesNH.y) + id_x * 3;
     int incrementPerImage = id_z * 3;
     d_float8 cmnParamsR_f8, cmnParamsG_f8, cmnParamsB_f8;
```

Comment:
Same comment for hip. Pls add a int3 cmnParamsLocs = make_int3(incrementPerImage, incrementPerImage + 1, incrementPerImage + 2);